lib/rialto/etl/logging.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rialto/etl/loaders/sparql'
+require 'yell'
+
+module Rialto
+  module Etl
+    # A module to hold a logger
+    module Logging
+      def logger
+        @logger ||= Yell::Logger.new(:null).tap do |logger|
+          # Everything to stderr
+          logger.adapter :stderr
```

Comment:
I wonder if we could/should write warn/error/fatal to `stderr` and anything less severe than warn to `stdout`. Or does using both stdout and stderr confuse things? cc: @justinlittman @tingulfsen @aaron-collier

Reply:
Current usage from the README for this repository suggest that you use e.g.

`exe/transform call StanfordOrganizations -i organizations.json > organizations.sparql`

in order to generate the output file. If we expect to continue with this, then I think it makes sense to use `stderr` for logging, since we'll otherwise mix "regular" output with logging output?

Reply:
Oh, yes, very good point. We don't want that.